pulsar-function-go/pf/instanceConf.go

```diff
@@ -42,6 +42,11 @@ type instanceConf struct {
 	killAfterIdle               time.Duration
 	expectedHealthCheckInterval int32
 	metricsPort                 int
+	authPlugin                  string
+	authParams                  string
+	tlsCertsPath                string
```

Comment:
Nit: I think we want this to be `tlsTrustCertsPath`. In the project, we typically use `tlsCertsPath` to be a server's public certs that get sent to the client.